ppr-ui/src/utils/ppr-api-helper.ts

```diff
@@ -89,6 +91,29 @@ export async function search (
     })
 }
 
+// Submit a search query (search step 1) request.
+export async function staffSearch (
+  searchCriteria: SearchCriteriaIF,
+  staffPayment: StaffPaymentIF,
+  certified: boolean
+): Promise<SearchResponseIF> {
+  let extraParams = '?'
+  // do they want a certified search
+  if (certified) {
+    extraParams = extraParams + 'certified=True'
+  }
+  // filled out staff payment parameters
+  if (staffPayment) {
+    if (extraParams.length > 0) {
+      extraParams = extraParams + '&'
+    }
+    extraParams = extraParams + 'routingSlipNumber=' + staffPayment.routingSlipNumber
+    extraParams = extraParams + '&bcolAccountNumber=' + staffPayment.bcolAccountNumber
+    extraParams = extraParams + '&datNumber=' + staffPayment.datNumber
```

Comment:
The parameters are conditional on payment method:
If FAS payment method only include routingSlipNumber.
If BCOL payment only include bcolAccountNumber and datNumber.
If both a routing slip and a bcol account number are submitted the API will return an error.

Reply:
Made this change